src/NServiceBus.Persistence.DynamoDB.AcceptanceTests/SetupFixture.cs

```diff
@@ -22,21 +22,10 @@ public async Task OneTimeSetUp()
 
             DynamoDBClient = client;
 
-            var installer = new Installer(new DynamoDBClientProvidedByConfiguration
-            {
-                Client = DynamoDBClient
-            }, new InstallerSettings
-            {
-                SagaTableName = TableName,
-                Disabled = false,
-                CreateOutboxTable = true,
-                CreateSagaTable = true
-            }, new OutboxPersistenceConfiguration
-            {
-                TableName = TableName
-            });
+            var installer = new Installer(DynamoDBClient);
 
-            await installer.Install();
+            await installer.CreateOutboxTableIfNotExists(new OutboxPersistenceConfiguration { TableName = TableName });
+            await installer.CreateSagaTableIfNotExists(new SagaPersistenceConfiguration { TableName = TableName });
```

Comment:
I think this shows a mismatch on the installer API because when we use a shared table concept, there is no need to also create the saga table really. I wonder whether we should change the installer to work with a generic `CreateTable(TableConfiguration)` API instead and then handle split table/shared table setups on higher-level APIs.